### PR TITLE
Reworked unit tests. Added dependency rewire. Added src/lib/enum.js

### PR DESCRIPTION
**Pull request by ggrg1**
Worked on unit tests.

### CODE REVIEW

File: src/lib/enum.js [hunk 1]
```diff
@@ -0,0 +1,310 @@
+/*****
```
Comment (ggrg) [inline src/lib/enum.js]: Can't see why adding this file was necessary.

File: test/unit/handlers/lib/utility.test.js [hunk 2]
```diff
@@ -0,0 +1,343 @@
...
+  })
+
+  utilityTest.afterEach(test => {
+    sandbox.restore()
+    test.end()
+  })
+
+  
+
+  // utilityTest.test('createParticipantTopicConf should', createParticipantTopicConfTest => {
```
Comment (ggrg) [inline test/unit/handlers/lib/utility.test.js]: Commented out code should be removed if unnecessary.

Comment (ggrg) [reply test/unit/handlers/lib/utility.test.js]: Applies to all files.

Comment (ggrg1) [reply test/unit/handlers/lib/utility.test.js]: Should I delete unused functions from src/handlers/lib/utility.js as well?

File: src/lib/enum.js (hunk 1 above)
Comment (ggrg1) [reply src/lib/enum.js]: Required in utility.test.js->createGeneralTopicConfTest.test unless the test itself is unnecessary.

Comment (ggrg) [reply src/lib/enum.js]: Is it?

File: test/unit/handlers/lib/utility.test.js (hunk 2 above)
Comment (ggrg) [reply test/unit/handlers/lib/utility.test.js]: If they are commented out - yes.

**Pull request merged**